suite-native/tokens/src/tokensSelectors.ts

```diff
@@ -163,6 +163,11 @@ export const selectAccountTransactionsWithTokenTransfers = createMemoizedSelecto
         ) as WalletAccountTransaction[],
 );
 
+export const selectHasAccountAnyTransactions = createMemoizedSelector(
+    [selectAccountTransactionsWithTokenTransfers],
+    (transactions): boolean => transactions.length > 0,
```

Comment:
What will the return value be for e.g. an ETH account without base transactions but some token transactions?

Reply:
OK, it seems that the base account shows all the transactions including token transactions. Never mind.